### ISSUE TITLE
Clock pulse missing on tempo changes in Reaper

### ISSUE THREAD

**Issue by taji** [DISTRHO/Cardinal#649]
### Version

24.04

### Plugin type

VST3

### OS

Linux (PopOs 20.04)

### DAW / Host (if applicable)

Reaper 6.34

### Description

To reproduce (see the attached Reaper project file:

1. Launch Reaper
2. Add a track | Insert new Cardinal vst3 plugin instance (I used the multi channel version)
3. In Cardinal, insert the following modules:  Host Time & Host Audio
4. Wire the Step output from the Host Clock module to the Host Audio module's input pin 1.

![image](https://github.com/DISTRHO/Cardinal/assets/702031/b225454f-5e40-4970-b300-ad56f65296ce)

5. Right click on the track and select Render/freeze tracks | Render tracks to Mono stem tracks

![CardinalMissingClockPulse-Issue2](https://github.com/DISTRHO/Cardinal/assets/702031/cabd18e9-45fe-4d9a-afe4-327c41ef7bba)

6. View the rendered audio clip at the 60 bpm and 120 bpm tempo changes.

EXPECTED:  Both tempo change points should have a clock pulse firing at beat 1.
ACTUAL:  The 60 BPM has the clock pulse, but the 120 BPM does not.  

![CardinalMissingClockPulse-Issue1](https://github.com/DISTRHO/Cardinal/assets/702031/1c61b545-6b6a-46ab-941b-76497bfde157)

NOTE:  Issue is not happening in VCV:

![image](https://github.com/DISTRHO/Cardinal/assets/702031/7a97c3dd-9527-4a87-9467-0c85624e2585)

**Comment by taji** [comment]
Here's the rpp file for reproducing in Cardinal.
[Reaper-Cardinal-ClockPulseBug.RPP.zip](https://github.com/DISTRHO/Cardinal/files/15014576/Reaper-Cardinal-ClockPulseBug.RPP.zip)

**Comment by falkTX** [comment]
thanks for the great report!

if you can please check if the same behaviour happens with other formats. so clap, lv2 and vst2.
that will help a lot in understand the issue better

**Comment by taji** [comment]
Yes, will do.  But it may be early next week as I'm out of touch this weekend.

**Comment by taji** [comment]
I'm returning to this to test CLAP, VST2 and LV2.  I can't get CLAP plugins to load.  I copied Cardinal.clap folder to ${HOME}/.clap.   Reaper scans the folder successfully.   But  when I load the CLAP version of Cardinal in Reaper, I'm seeing this error message: 

![image](https://github.com/DISTRHO/Cardinal/assets/702031/788a3335-6935-4de3-a7cd-6d7ded8fd1c6)

Do I need to install other files?  Maybe the standalone application goes in /usr/local/share/cardinal?

TIA

**Comment by taji** [comment]
Testing LV2 plugin, I'm seeing the inverse of the VST3 issue:  at the 60 BPM we do NOT see the clock pulse event but at 120 bpm we do.  

![image](https://github.com/DISTRHO/Cardinal/assets/702031/d33d76cd-d813-470b-96de-4318160a6c95)

**Comment by taji** [comment]
Testing VST2 fx plugin.  I see the same missing spike at the 120 bpm marker:

![image](https://github.com/DISTRHO/Cardinal/assets/702031/57fb0ab7-2384-4b87-a234-c1b948bc1828)

**Comment by taji** [comment]
@falkTX :  I see my problem is a duplicate of the issue here: https://github.com/DISTRHO/Cardinal/issues/583

I'll monitor the discussion and test once a solution is provided.
